### PR TITLE
Turn forum toolbar icons white in dark mode

### LINKED ISSUE #5923: Change non-material forum editor buttons to light versions when a dark theme is enabled
### Why this would be helpful

With the dark-www addon, it'd be nice for buttons to not have a light background, however dark-www makes buttons have one as otherwise, the buttons would have very low contrast.

### My suggestion

When dark-www has a dark theme, replace the icons with light alternatives instead.

### Possible alternatives

- An option in material-forum-editor-buttons, which is what this issue was at first, however, it makes more sense to have this built into the dark-www addon as it's something that is expected to come with dark mode.

### Additional context

None.

### PR DESCRIPTION
**Pull request by mybearworld**
Resolves #5923

### Changes

Remove the background of the forum toolbar icons in dark mode, and instead replace them with white alternatives.

### Reason for changes

To use dark buttons in dark mode.

### Tests

<s>Tested locally on Edge 114.0.1788.0 (Official build) dev (64-bit)</s> I'm not sure how to download branches, so I can't test it properly.

### CODE REVIEW

**Comment by mybearworld** [comment]
This is the first time I have done something to the code of Scratch Addons besides adding a line to a CSS file, so I might have done something wrong. I hope not, though!

File: addons/dark-www/addon.json [hunk 1]
```diff
@@ -1863,6 +1863,114 @@
         },
         "a": 0.74
       }
+    },
+    {
+      "name": "forum-bold-button",
+      "value": {
+        "type": "textColor",
+        "black": "url(https://cdn.scratch.mit.edu/scratchr2/static/__e65d8a7c03267938d906e22792610478__//djangobb_forum/js/markitup/sets/bbcode/images/bold.png)",
+        "white": "url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAhFBMVEUAAADe3t7T09POzs7s7Ozg4ODn5+fh4eHY2Njz8/PT09P19fX7+/vv7+/c3Nz4+PjV1dXOzs7g4ODa2trLy8v39/fu7u7s7Ozo6OjT09Po6Ojm5ub6+vr4+Pjl5eXY2Njl5eXLy8vu7u7h4eHj4+Pd3d3a2trX19fS0tLz8/Pm5ubMzMxZSeJAAAAAInRSTlMAf79/z8+/v5+Pf3AgEBDv7+/f38+/v7+vr5+fj39/f2BgO3gytQAAAGJJREFUGNOdykUOgDAARNEp7u5Ocbj//SAEGtj27+ZlwFcfHb6Ye5rDRFwFQJvlP5BpZGAsApxgKD8QEnlQwGpnFSh2yX6BTBfYlJoM7vtG6xeaUQJM6lrP7rI00fW4ssDXCeLqBY1pf6pbAAAAAElFTkSuQmCC)",
```
Comment (Samq64) [inline addons/dark-www/addon.json:1872]: Cool solution, but the addon.json is already really big. Could separate PNG files be used instead?

Comment (mybearworld) [reply addons/dark-www/addon.json:1872]: I have already tried that, but neither %addon-self-dir%/assets/file.png nor ./assets/file.png worked, so I went with this instead.

Comment (mxmou) [reply addons/dark-www/addon.json:1872]: The value of a `customCssVariable` can reference other CSS variables. So you can set something like this in the CSS file:
```css
:root {
  --darkWww-forumBoldButtonBlack: url("https://cdn.scratch.mit.edu/scratchr2/static/djangobb_forum/js/markitup/sets/bbcode/images/bold.png");
  --darkWww-forumBoldButtonWhite: url("%addon-self-dir%/assets/forum_buttons/bold_white.png");
}
```

And this in the `addon.json`:
```json
"black": "var(--darkWww-forumBoldButtonBlack)",
"white": "var(--darkWww-forumBoldButtonWhite)",
```

Comment (mybearworld) [reply addons/dark-www/addon.json:1872]: That's a lot better than my solution, I'll do that.

Comment (mybearworld) [reply addons/dark-www/addon.json]: Done, but I'm not sure how I'd test it now because I don't know how to download a branch.

**Comment by mybearworld** [comment]
Oh great, the Prettier check is failing, I can't figure out what I did wrong from the message, though.

**Review by mxmou — APPROVED** [review]
Tested on Edge and Firefox.

**Review by DNin01 — APPROVED** [review]
Tested on Edge 112. Looks okay to me!

**Pull request merged**